### ISSUE TITLE
is this still being developed?

### ISSUE THREAD

**Issue by r3wt** [Stanko/rocketScroll#2]
If you don't plan to maintain it, would you consider electing a new maintainer? I'd love to take over it.

I just wrote a basic directive to use it in angular. it works, but there are no visible scrollbar:

```js
.directive('ngRocketScroll',function($timeout){
	return {
		restrict: 'A',
		link: function(scope,element){
			var instance = new RS.RocketScroll(angular.element(element)[0],true);
			var timeout = null;
			scope.$watch(
				function(){
					return element;
				},
				function(){
					if(timeout != null){
						$timeout.cancel(timeout);
						timeout = null;
					}
					$timeout(function(){
						instance.refresh();
					},100);
				}
			);
		},
		transclude: true,
		template: '<div class="scrollDiv"><div class="scrollContent" ng-transclude></div>'
	}
})
```
usage:
```html
<div ng-rocketscroll>
//content here.
</div>

**Comment by r3wt** [comment]
@Stanko it works good, one thing is the scrollbar never displays, because it incorrectly calculates size of div and decides the inner height is smaller than height of container. i'm thinking its probably a race. despite the scrollbar being fucked and not displaying, it works fine. scrolls completely.

**Comment by Stanko** [comment]
I answered on the other issue, before I got to this one, it seems you resolved it already :)

To be honest, library is not in the active development anymore. I have plans to rewrite it, and to provide react component as well, but no real time frame to do it. So, I would be happy to give you a write permission, I just sent an invite.

As for the scroll bars not showing up, I will take a look, but I'm not using `angular 1.x` for some time now, so I can't promise much.

Thank you so much for the effort!

**Comment by r3wt** [comment]
@Stanko cool man. I'm excited to take over. i'll be good to your baby, promise :)

**Comment by r3wt** [comment]
@Stanko to be honest man, i've completely rewritten nearly, other than some of the basic code. it will break backwards compatibility? are you ok with that, or should i just make my own fork and call it like "RocketScroll2" and leave a note that you wrote the original code.

**Comment by Stanko** [comment]
@r3wt as you wish :) feel free to fork if you want the ownership and the credit for it. I'm also fine with keeping it in this repository.

Anyway, I'm looking forward to see the new version!

Thanks for the effort!

**Comment by r3wt** [comment]
@Stanko it's up here https://github.com/r3wt/RocketScroll for now. havent published to bower yet.

**Comment by Stanko** [comment]
Awesome! Can I suggest adding `gh-pages` branch with a demo?

**Comment by r3wt** [comment]
@Stanko yes, i'll get to that at some point. as of now, i have 3 different demos:

- angular directive
- jquery plugin
- plain js version

I'd like to add react, but i don't use react. Maybe if you are still interested in the project you could contribute the react wrapper and demo page for its useage. Up to you, if you don't have the time, i could probably stumble my way through it, but since I'm not a react dev it would probably be *not* optimal.
